src/mixins/_fluid.scss

```diff
@@ -37,7 +37,7 @@
 }
 
 /**
- * Fluid font-size
+ * Fluid properties
```

Comment:
IMO: This would benefit from a comment explaining benefits or common use cases of these fluid properties.

Reply:
What would this comment say that the earlier comments in the file do not?

Reply:
I see...Because of the way the file was presented in the GitHub :Files Changed" interface, I did not see the preceding comments.